pkg/smartcontract/rpcbinding/binding.go

```diff
@@ -162,9 +176,9 @@ type ContractReader struct {
 	{{end -}}
 	invoker Invoker
 }
-
```

Comment:
Mixing spacing changes and logic ones in one commit leaves some space for improvement.

Reply:
Agree, I moved a part which fixes the indentation to a separate commit.